common-test/src/main/java/io/strimzi/test/Topic.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation for test classes or methods run via {@code @RunWith(StrimziRunner.class)}
+ * which causes that runner create kafka config map with Topic before,
+ * and delete kafka config map with Topic after the tests.
+ */
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Topic {
```

Comment:
Sorry @asorokhtey, I recommended to you don't use repeatable for `@Topic` annotation, but after thinking a little bit more about where we can use it, I imagined a few test cases when we need to have like 2 or more topics.
Thus, can you plz revert to your first implementation? sorry again.

```
@Topic(name = "test-topic-1", clusterName = "my-cluster")
@Topic(name = "test-topic-2", clusterName = "my-cluster")
    public void testSendMessages() {...}
```